### DISCUSSION TITLE
Invitation list management

### DISCUSSION THREAD

**Discussion by eladiogarcia** [opengisch/QField#2458] [Q&A]
I am Eladio García and I am trying to add to 2 persons to my project for testing the app. One alredy accepted my invitation but I have a problem, as one of them is not receiving emails with the invitation (I don't know why- He has already checked SPAM filter. 
I tried to send it again to another email, but now is added to the waiting list until QFieldCloud is officially released.
So, I wonder if is posible to manage invitations sent. i.e. delete it from the list of people invited to QFieldCloud by me
![Captura de pantalla 2021-10-27 141028](https://user-images.githubusercontent.com/93260863/139062751-47c29377-7892-42e6-8bae-a12d2711b383.png)

Best regards

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@eladiogarcia Hey Eladio, is this still an issue? If yes, they can directly register on QFieldCloud now without an invitation. ;) Just go to the login screen and the "register" button is awaiting them.

**Comment by eladiogarcia** [comment]
Dear Ivan,
Thanks for your support. We solved it resending invitation from another member if I remember well. But we haven’t tried it again since that.
As I can understand from your email, is QFieldCloud registration free at this moment? (i.e. without invitation)
We are still testing the app in one project with a remote collaborator, trying to set the right workflow for editing and synchronizing data, but waiting for definitive deployment to generalize its use in our team.
Best wishes,

Eladio L. García de la Morena

Co Director
+34 666 97 39 56
***@***.******@***.***>
www.biodiversitynode.com<http://www.biodiversitynode.com/>
***@***.***


De: Ivan Ivanov ***@***.***>
Enviado el: miércoles, 9 de marzo de 2022 8:23
Para: opengisch/QField ***@***.***>
CC: Eladio L. García de la Morena ***@***.***>; Mention ***@***.***>
Asunto: Re: [opengisch/QField] Invitation list management (Discussion #2458)


@eladiogarcia<https://github.com/eladiogarcia> Hey Eladio, is this still an issue? If yes, they can directly register on QFieldCloud now without an invitation. ;)

—
Reply to this email directly, view it on GitHub<https://github.com/opengisch/QField/discussions/2458#discussioncomment-2321963>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWHQYP5T74TGKQ6DHHFXJZDU7BGVPANCNFSM5NJEPQSA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by suricactus** [reply]
Yes, in the next month you can freely register and use the full functionality of QFieldCloud for free.
